### PR TITLE
Integrate docs with jekyll

### PR DESCRIPTION
**Pull request by ratik21**
- User docs are available in `/docs/guide`
- Updated api docs to generate in `/docs/api/`
- Display User docs using jekyll theme (minimal)
- Font of user docs is same as landing page

User docs look like:
image
image
image
image

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+# These are used to personalize your new site. If you look in the HTML files,
+# you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
+# You can create any custom variable you would like, and they will be accessible
+# in the templates via {{ site.myvariable }}.
+
+title: "Algo Builder"
+name: "Algo Builder"
+baseurl: "/algo-builder" # the subpath of your site, e.g. /blog
+url: "https://scale-it.github.io" # the base hostname & protocol for your site, e.g. http://example.com
+repository: "scale-it.github.io/algo-builder"
```
Comment (robert-zaremba) [inline docs/_config.yml:25]:
```suggestion
url: "https://scale-it.github.io/algo-builder/" # the base hostname & protocol for your site, e.g. http://example.com
repository: "https://github.com/scale-it/algo-builder"
```

File: docs/_config.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+minimal_mistakes_skin: "air" # "air", "aqua", "contrast", "dark", "dirt", "neon", "mint", "plum" "sunrise"
+
+# Build settings
+markdown: kramdown # markdown processor
+remote_theme: "mmistakes/minimal-mistakes@4.22.0"
+
+plugins:
+  - jekyll-feed
```
Comment (robert-zaremba) [inline docs/_config.yml:34]: is this plugin needed?

File: docs/_data/navigation.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+main:
+  - title: "Quick-Start Guide"
+    url: https://github.com/scale-it/algo-builder#installation
+  - title: "API Docs"
+    url: https://scale-it.github.io/algo-builder/ # TODO: Update to custom api readme link
```
Comment (robert-zaremba) [inline docs/_data/navigation.yml:5]: please create a task for this in pivotal tracker

File: docs/_data/navigation.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+  - title: "Index"
+    url: /guide/README
```
Comment (robert-zaremba) [inline docs/_data/navigation.yml:7]: Please add links to:
-  _Home_ (https://scale-it.github.io/algo-builder/)
- _Dapp Templates_ repository


The order of the links should be: Home, Quick Start, Guide, API,

File: docs/_data/navigation.yml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+main:
+  - title: "Quick-Start Guide"
+    url: https://github.com/scale-it/algo-builder#installation
+  - title: "API Docs"
```
Comment (robert-zaremba) [inline docs/_data/navigation.yml:4]:
```suggestion
  - title: "Quick-Start"
    url: https://github.com/scale-it/algo-builder#installation
  - title: "API"
```

File: docs/_data/navigation.yml [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+    url: https://scale-it.github.io/algo-builder/ # TODO: Update to custom api readme link
+  - title: "Index"
```
Comment (robert-zaremba) [inline docs/_data/navigation.yml:6]:
```suggestion
  - title: "Guide"
```

**Review by robert-zaremba — APPROVED** [review]
Good job @ratik21 . Preapproving.
- left few suggestions and comments

File: docs/_config.yml (hunk 2 above)
Comment (ratik21) [reply docs/_config.yml:34]: No, it was added by default. I don't think we can remove it - [https://github.com/github/pages-gem/issues/627](https://github.com/github/pages-gem/issues/627)
For now i have commented it, with link to the open issue

File: docs/_data/navigation.yml (hunk 3 above)
Comment (ratik21) [reply docs/_data/navigation.yml:5]: OK.

File: docs/_data/navigation.yml (hunk 4 above)
Comment (ratik21) [reply docs/_data/navigation.yml]: Done.  BDW - The title in top bar (`Algo Builder` on LHS) also points to HOME (`/`)

File: docs/_config.yml (hunk 1 above)
Comment (ratik21) [reply docs/_config.yml:25]: - Updating URL as per suggested name (but not sure, as base url is supposed to have /algo-builder suffix)
- Cannot update the repository url (the correct format is `user/repo`). So i think it should be `scale-it/algo-builder` (removing `github.io`). Updating to given suggestion gives:
image

Comment (robert-zaremba) [reply docs/_config.yml]: I see - OK, let's use what they suggest.

**Pull request merged**